packages/datadog-plugin-aerospike/src/index.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+const { storage } = require('../../datadog-core')
+const DatabasePlugin = require('../../dd-trace/src/plugins/database')
+
+class AerospikePlugin extends DatabasePlugin {
+  static get id () { return 'aerospike' }
+  static get operation () { return 'command' }
+  static get system () { return 'aerospike' }
+
+  bindStart (ctx) {
+    const { commandName, commandArgs } = ctx
```

Comment:
Should the arguments be normalized on the instrumentation side? My worry is that not only commands clearly have different arguments, but they could also differ between versions.

Reply:
shouldn't they be normalized on the subscriber side? since the instrumentation is just attaching every argument the command gets to the ctx & then the subscriber decides what to do with that?

Reply:
What I meant is that the relevant information could be extracted from the publisher into a normalized data structure that contains the information in a way that is guaranteed not to change between versions. This has the benefit of the subscriber never having to worry about changes in the arguments, and having to handle the same normalization logic in every subscriber.

However, since both the publisher and subscriber lives in the same repo, it might not be worth it to tackle this right now.